src/components/PersonItem/PersonItem.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import classNames from 'classnames';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonItem: React.FC<Props> = ({ person }) => {
+  const { slug } = useParams();
+
+  const GENDER_FEMALE = 'f';
+  const { name, sex, born, died, motherName, fatherName, mother, father } =
+    person;
+
+  return (
+    <tr
+      data-cy="person"
+      key={person.slug}
+      className={classNames({
+        'has-background-warning': person.slug === slug,
+      })}
+    >
+      <td>
+        <PersonLink
+          to={`../people/${person.slug}`}
+          className={classNames({
+            'has-text-danger': sex === GENDER_FEMALE,
+          })}
```

Comment:
`Person` link should accept the whole person and manage slug/classname/name itself
![image](https://github.com/mate-academy/react_people-table-basics/assets/63243136/0aba2539-baaf-4ecb-a1da-f733bbf29918)